selfdrive/car/mazda/interface.py

```diff
@@ -57,7 +57,9 @@ def get_params(candidate, fingerprint=gen_empty_fingerprint(), car_fw=None, disa
       ret.lateralTuning.pid.kpV, ret.lateralTuning.pid.kiV = [[0.19], [0.019]]
       ret.lateralTuning.pid.kf = 0.00006
 
-    if candidate not in (CAR.CX5_2022, ):
+    if candidate in (CAR.CX5_2022, ):
```

Comment:
@Jafaral should this also include CX-9 2021?

Reply:
No it shouldn't. CX-9 2021 still has the 28mph min steer speed.